acoustid/future/fpindex/client.py

```
@@ -0,0 +1,282 @@
+from typing import Any, Dict, List, Optional, Tuple, Union
+from urllib.parse import urljoin
+
+import aiohttp
+
+logger = logging.getLogger(__name__)
+
+
+class FingerprintIndexClientError(Exception):
+    """Base exception for fingerprint index client errors"""
+
+    pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)